literature.bib

```diff
@@ -53,7 +63,7 @@ @inproceedings{HST+:MODELS22
     address = Washington,
 	month = OCT,
     year = 2022,
-    note = {To appear}
+	typo3Tags = {FMCounting}
 }
 
```

Comment:
Did you move the entries that lost their "to appear"-note to the chunk of papers in 2022? (Starting with `%%%%%%%%%%%%%%%% 2022 %%%%%%%%%%%%`)

Reply:
Good catch, I now moved them to the correct chunk.